test/functional/mempool_fee_histogram.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test mempool fee histogram."""
+
+from decimal import Decimal
+
+from test_framework.blocktools import COINBASE_MATURITY
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_greater_than,
+    assert_greater_than_or_equal,
+)
+
+class MempoolFeeHistogramTest(BitcoinTestFramework):
+    def add_options(self, parser):
+        self.add_wallet_options(parser)
+
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        node = self.nodes[0]
+        self.generate(self.nodes[0], COINBASE_MATURITY + 2, sync_fun=self.no_op)
+
+        # We have two UTXOs (utxo_1 and utxo_2) and we create three changeless transactions:
+        # - tx1 (5 sat/vB): spending utxo_1
+        # - tx2 (14 sat/vB): spending output from tx1
+        # - tx3 (6 sat/vB): spending utxo_2 and the output from tx2
+
+        self.log.info("Test getmempoolinfo does not return fee histogram by default")
+        assert("fee_histogram" not in node.getmempoolinfo())
```

Comment:
> missing whitespace after keyword
```suggestion
        assert ("fee_histogram" not in node.getmempoolinfo())
```

Reply:
> btw: It would be nice to have a lint warning about this. If it was not added since my last push.

There was, the lint job failed with this error. I just copy-pasted it.